### ISSUE TITLE
Juices should be drunk from a glass bottle

### ISSUE THREAD

**Issue by lawremi** [iguanaman/HungerOverhaul#8]
Would be neat if e.g. Pam's fruit juices would require a glass bottle in the recipe and would yield a bottle filled with juice. Upon drinking, hunger is restored. Would require registering the juices as forge fluids, which is a good idea anyway. Let me know if you want me to contribute this.

**Comment by iguanaman** [comment]
Sounds interesting but I'm a little wary of making changes to the recipe structure.  Without NEI it's hard to keep track of all the recipes and having HO change them would only make things worse.  What would be the actual gameplay benefit of doing this do you think?

**Comment by lawremi** [comment]
I agree we should make the recipe changes in PHC itself. How receptive is
Pam to these changes? She hasn't responded to any of my queries, public or
private.

**Comment by iguanaman** [comment]
She hasn't update the mods in a while and can only assume she is on hiatus, how long ago did you send those messages to her?  

Are you on IRC?  Jump in #iguanamods sometime
